ckanext/external_storage/tests/test_actions.py

```diff
@@ -0,0 +1,72 @@
+from ckan.tests import factories
+from ckan.tests import helpers as test_helpers
+
+from ckanext.external_storage.tests import get_context
+
+
+class TestBlobSotrageActions():
```

Comment:
With `pytest`, there is really almost never a good reason to put tests in a class. With modern fixtures, this is typically just an overhead that is not required and may cause issues